amplify/backend/function/deleteUser/src/index.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import gremlin_python
+from gremlin_python.driver import client, serializer
+
+def lambda_handler(event, context):
+    endpoint = os.environ['NEPTUNE_ENDPOINT']
+    port = os.environ['NEPTUNE_PORT']
+
+    # Connect to Neptune
+    db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
```

Comment:
I believe its better to establish database connections outside of the Lambda function handler, see this [AWS doc](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html) and [blog post](https://www.qloudx.com/pool-share-database-connections-in-lambda-functions-use-rds-proxies/#:~:text=The%20standard%20best%20practice%20when,invocations%20of%20a%20Lambda%20function.).
```suggestion
endpoint = os.environ['NEPTUNE_ENDPOINT']
port = os.environ['NEPTUNE_PORT']

# Connect to Neptune
db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
def lambda_handler(event, context):
```